### ISSUE TITLE
Checkbox border has insufficient contrast against its adjacent color in "Enable telemetry dialogue"

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#854]
**Describe the bug**
The border of the checkbox in "enable telemetry dialogue" fails Non-text Contrast requirement as it has insufficient contrast against its adjacent background

The following AI-windows mode(s) have this issue:
- Regular mode
- High Contrast White mode

**Expected behavior**
The border of the checkbox should have a contrast ratio of at least 3:1 against its adjacent color

**Actual behavior**
The non-text contrast ratio of the checkbox's border against its background fall short of 3:1

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![1 1](https://user-images.githubusercontent.com/22068434/88341240-184be380-ccf2-11ea-8d04-12c9c3105ff1.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
Bug was originally reported against light mode, but I'm getting a contrast ratio of 4.88 in light mode, which satisfies the requirement. The contrast in HC White is 1.8, which fails the requirement. All modes are satisfied after the change.

**Comment by DaveTryon** [comment]
@iamrafan, this fix will be in release 1.1.1370.01 (will release October 2nd). Please regress and close once the build is out.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1380.01, the checkbox border is now compliant in all 6 modes.
